src/popup/pages/Send/index.js

```diff
@@ -217,16 +276,25 @@ class SendPage extends React.Component {
     if (account.address) {
       this.props.updateNetAccount(account)
     }
+    if(!isSilent){
+      Loading.hide()
+    }
   }
 
   componentWillReceiveProps(nextProps) {
     if (nextProps.refreshAccountCommon && !this.isRequest) {
-      let address = this.props.currentAccount.address
-      this.fetchData(address)
+      let { isWithdraw } = this.pageConfig
+      if(isWithdraw){
+        this.fetchParatimeData(true)
+      }else{
+        this.fetchData(true)
```

Comment:
Missing await?

Reply:
no , there does not need await , just silent update ，if use await here . may cause block process .